src/util.js

```diff
@@ -91,6 +91,17 @@ export function getCourseInfo(courseName) {
   return courseInfo.content;
 }
 
+export function getLessonIntro(lesson) {
+  let lessonContent = require('onlyContent!lessonSrc/' + lesson + '.md').content;
+  lessonContent = lessonContent.substring(lessonContent.indexOf('<section class="intro"'));
+  let text = lessonContent.substring(lessonContent.indexOf('<p>'), lessonContent.indexOf('</p>'));
+  let picture = lessonContent.substring(lessonContent.indexOf('<img'), lessonContent.indexOf('</figure'));
```

Comment:
Dersom `indexOf` ikke finne det den leter etter, så gir den tilbake `-1`. Og når substring får `-1`, så later den som om det er `0`. Forøvrig, dersom start-index > end-index, så bytter de bare plass. Så dersom den finner start-index (f.eks. 5) men ikke end-index (som da blir -1), så blir det gjort om til `substring(0,5)`. Er det det vi vil ha? Kanskje legge inn noe sjekker for at vi fikk det vi ville? Evt bare bruke regex? Dersom du vil teste med litt regex, så er det mange websider som har en "regex online tester"-greie for å sjekke at du skriver riktig regex...